tensorflow/python/ops/math_ops.py

```diff
@@ -100,6 +100,7 @@
 @@complex
 @@conj
 @@imag
+@@arg
```

Comment:
But then we would have some functions in the root namespace and others in math. Other than being less frequently used is there a reason to move to a namespace separate from other related functions like abs, conj etc?